bigtable/bigtable.sh

```diff
@@ -243,6 +261,9 @@ function install_hbase() {
 }
 
 function main() {
+  if [[ ${OS_NAME} == debian ]] && [[ ${DATAPROC_IMAGE_VERSION} == 2.0 || ${DATAPROC_IMAGE_VERSION} == 1.5 ]]; then
```

Comment:
I think 2.1 should not be excluded.  Maybe 2.2 since we changed the format and filename of the archive list, but 2.1 will eventually be oldstable and we'll want the code to work there, too.